build/goimports.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename '.github' \
+        -o -wholename './build/_workspace' \
+        -o -wholename './build/bin' \
+        -o -wholename './crypto/bn256' \
+        -o -wholename '*/vendor/*' \
+      \) -prune \
+    \) -name '*.go'
+}
+
+GOFMT="gofmt -s -w";
+GOIMPORTS="goimports -w";
+find_files | xargs $GOFMT;
+find_files | xargs $GOIMPORTS;
```

Comment:
What's the purpose of this file?

Reply:
One of the reasons of linting errors was due to having code not properly formatted and not properly imported. There was no standard set for that. Now there is.

Reply:
`goimports` is pretty much the standard tool used by any Go development environment. I don't really see the value of providing an extra utility method in our repo that is covered by the developer's editor anyway. I completely agree that the linter should enforce it, but its the developer's job to configure their dev environment, not ours.

Reply:
When I started to work on this issue, I found that the devs are not really doing what you said is their job... 
```
consensus/consensus.go:1::warning: file is not goimported (goimports)
swarm/fuse/fuse_dir.go:1::warning: file is not goimported (goimports)
swarm/fuse/swarmfs.go:1::warning: file is not goimported (goimports)
```

So I ran the goimports the way "I think" is the best way, splitting the imports in 3. Just to find out that this is not the proffered way here. 
Also, running `goimports ./..` throws some errors, that I'm not sure how to fix, or even if they are really errors, so I added the script:
```
goimports -w ./..
../go-ethereum/crypto/bn256/bn256_fast.go:26:9: expected type, found '='
../go-ethereum/crypto/bn256/bn256_fast.go:30:9: expected type, found '='
../go-ethereum/crypto/bn256/bn256_fast.go:34:2: expected declaration, found 'return'
../go-ethereum/crypto/bn256/bn256_slow.go:26:9: expected type, found '='
../go-ethereum/crypto/bn256/bn256_slow.go:30:9: expected type, found '='
../go-ethereum/crypto/bn256/bn256_slow.go:34:2: expected declaration, found 'return'
```
So I added a "default" script to get this sorted and leave no room for errors.